install/ansible/cri-containerd.yaml

```diff
@@ -0,0 +1,46 @@
+---
+- hosts: all 
+  become: true
+  # Setting the environment variables that might be needed for the entire installation
+  environment:
+    PATH: "{{ ansible_env.PATH }}:/opt/golang/bin:/usr/local/go/bin:/usr/local/bin"
```

Comment:
Do we still need gopath? Seems not.

Reply:
yes will remove it . I was using it for crictl build.